packages/sui-ssr/README.md

```diff
@@ -47,11 +48,26 @@ It will, over parameter, make that the express server run over a username and pa
     -C, --clean  Remove previous zip
     -h, --help   output usage information
     -A, --auth <username:password> Will build the express definition under authentication htpassword like.
+    -N, --outputFileName <outputFileName> A string that will be used to set the name of the output filename. Keep in mind that the outputFilename will have the next suffix <outputFileName>-sui-ssr.zip
```

Comment:
To me, adding the suffix makes no sense.

Reply:
The sufix was made in order to have 'tracked' which were the files that we created using the tool to delete on -C flag selected.